src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './Filter.scss';
+
+const Filter = ({ filters }: { filters: string[] }) => {
+  return (
+    <section className='filter'>
+      <div className='filter-container'>
+        <ul>
+          {filters.map((value) => (
+            <li>
+              <p>{value}</p>
+              <button type='button'>
+                <span>&#215;</span>
+              </button>
+            </li>
+          ))}
+        </ul>
+        <button type='button' className="clear">Clear</button>
```

Comment:
`Question`

`type='button'`? What's its purpose?

Reply:
You should always carry your attribute, it is good HTML practice